src/Microsoft.Identity.Web.TokenCache/TokenCacheErrorMessage.cs

```diff
@@ -6,5 +6,6 @@ namespace Microsoft.Identity.Web
     internal static class TokenCacheErrorMessage
     {
         public const string InitializeAsyncIsObsolete = "IDW10801: Use Initialize instead. See https://aka.ms/ms-id-web/1.9.0. ";
+        public const string ExceptionDeserializingCache = "IDW10802: Exception occurred while deserializing token cache. See https://aka.ms/msal-net-token-cache-serialization general guidance and https://aka.ms/ms-id-web/token-cache-troubleshooting for token cache troubleshooting information.";
```

Comment:
Is there an action the developer needs to take when they see this error message? It's not clear in the aka.ms links what the action should be when receiving this message.

Reply:
Updated now.